test/metabase/cmd/rotate_encryption_key_test.clj

```diff
@@ -0,0 +1,102 @@
+(ns metabase.cmd.rotate-encryption-key-test
+  (:require [clojure.java.io :as io]
+            [clojure.java.jdbc :as jdbc]
+            [clojure.string :as str]
+            [clojure.test :refer :all]
+            [metabase.cmd.load-from-h2 :as load-from-h2]
+            [metabase.cmd.rotate-encryption-key :refer [rotate-encryption-key!]]
+            [metabase.db :as mdb]
+            [metabase.db.connection :as mdb.connection]
+            [metabase.db.spec :as db.spec]
+            [metabase.driver :as driver]
+            [metabase.models :refer [Database Setting]]
+            [metabase.test :as mt]
+            [metabase.test.data.interface :as tx]
+            [metabase.util.encryption-test :as eu]
+            [toucan.db :as db]))
+
+(defn- persistent-jdbcspec
+  "Return a jdbc spec for the specified `db-type` on the db `db-name`. In case of H2, makes the connection persistent
+  10secs to give us time to fetch the results later."
+  [db-type db-name]
+  (case db-type
+    :h2 {:subprotocol "h2"
+         :subname     (format "mem:%s;DB_CLOSE_DELAY=10" db-name)
+         :classname   "org.h2.Driver"}
+    :postgres (db.spec/postgres (tx/dbdef->connection-details :postgres :db {:database-name db-name}))
+    :mysql (db.spec/mysql (tx/dbdef->connection-details :mysql :db {:database-name db-name}))))
+
+(defn- abs-path
+  [path]
+  (.getAbsolutePath (io/file path)))
+
+(defn- raw-value [key]
+  (:value (first (jdbc/query mdb.connection/*jdbc-spec*
+                             ["select value from setting where setting.key=?;" key]))))
```

Comment:
mysql required to add the tablename `setting` in the filter. Is it really needed? or maybe there's a config to not require it?

Reply:
Could it be because `value` and `key` are reserved words and would require backtick enclosure like `` `value` `` ?
https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Reply:
ah, that might be it. Thanks!